tests/ui/literals.stderr

```diff
@@ -22,7 +22,7 @@ error: integer type suffix should be separated by an underscore
   --> $DIR/literals.rs:16:27
    |
 LL |     let fail_multi_zero = 000_123usize;
-   |                           ^^^^^^^^^^^^
+   |                           ^^^^^^^^^^^^ help: add an underscore: `000_123_usize`
```

Comment:
Can you remove the tests you moved to the new file from the old file?

Reply:
The reason I left it there was that otherwise there wouldn't be a UI test of the leading zero plus unseparated suffix case. I guess it's not that important?

Reply:
Ah ok. Can you allow the `unseperated_literal_suffix` lint in the `literals.rs` test then?